modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc

```diff
@@ -210,10 +210,10 @@ image::shared:byoc_apply.png[cloud_byoc_apply]
 
 Redpanda Cloud does not support the following self-hosted functionality:
 
-- OpenID Connect (OIDC) authentication
+- Data transforms (currently in beta for Redpanda Cloud)
+- Remote Read Replicas (currently in beta for Redpanda Cloud)
+- OIDC authentication
 - Kerberos authentication
-- Data transforms
-- Remote Read Replicas
 - Redpanda Console topic documentation
 - Setting `auto_create_topics_enabled=true` for BYOC and dedicated clusters
```

Comment:
@Feediver1 seems like a peculiar specific callout since directly setting  ANY cluster configuration in cloud is still not supported.

cc @towfiqa

Reply:
We also specifically call this out in the note at the end of [this section for Serverless](https://docs.redpanda.com/current/deploy/deployment-option/cloud/serverless/#interact-with-your-cluster).

Reply:
hmm, ok I guess its a significant functional difference.  We may in fact enable it at some point in BYOC/Dedicated.

Thoughts @bpraseed ?